### DISCUSSION TITLE
How to use mdanalysis to deal with non-standard amino acid systems

### DISCUSSION THREAD

**Discussion by ldx022** [MDAnalysis/mdanalysis#4435] [Q&A]
Hello everyone! 
I've encountered some issues while processing my coarse-grained system with MDAnalysis. In general, my system is quite unique, not a very standard MD system, but a coarse-grained system I built manually. Overall, this system's topology files include only `.top`, `.gro`, and `.psf` files. However, I previously tried reading the `.top` format topology file with MDAnalysis and failed (I guess it might be due to non-standard amino acid names?). The `.psf` file seems like doesn't pair with the `.xtc` trajectory (although in VMD, the `.xtc` trajectory can load the `.psf` topology file for visualization, but not in MDAnalysis), so only `.gro` is used as the topology file. Could you please tell me MDAnalysis reads` .xtc` trajectories, what would be the most appropriate format for the accompanying topology file? How can MDAnalysis be used to solve problems like the ones I'm experiencing?

**Comment by hmacdope** [comment]
Hi @ldx022, you should be able to pair any topology format with any trajectory format basically, with the only constraint that the number of particles in each matches. For your use case (sounds like you used GROMACS) using a .gro and a .xtc file sounds like a good option.  MDAnalysis does read .xtc files. The topology parsing from a `.top` GROMACS format include file is more complicated and can sometimes not work as intended .

```python
import MDAnalysis as mda
u = mda.Universe("system.gro", "trajectory.xtc")
```
should work.
